god-mode.el

```diff
@@ -134,7 +146,9 @@ For example, calling with arguments 5 and t yields the symbol `S-f5'."
 (define-minor-mode god-local-mode
   "Minor mode for running commands."
   :init-value nil
-  :lighter " God"
+  :lighter (god-mode-lighter-string
+	    ((" "
+	     (:propertize god-mode-lighter-string face god-mode-lighter))))
```

Comment:
Minor: indentation seems off here. Use spaces instead of tabs, or simply `M-x untabify` this region.